app/views/admin/failed_bikes/index.html.haml

```diff
@@ -21,7 +18,7 @@
       %th
         Errors
     %tbody
-      - @b_params.each do |b_param|
+      - @b_params.includes(:creator).each do |b_param|
```

Comment:
Why not do this `includes` in the controller? That's what we've done in other places in the app. If there's a good reason to do it in the view instead, let's do it (and add an issue for moving the other ones into the views)